packages/Input/src/index.tsx

```diff
@@ -43,17 +43,25 @@ const Input: FunctionComponent<IInputProps> = ({
   utils: { classNames, ErrorMessage },
   ...attrs
 }): JSX.Element => {
-  const [newValue, setNewValue] = useState<string | undefined>(value);
+  const isValidValue = value !== null && value !== undefined;
+
+  const [newValue, setNewValue] = useState<string | undefined>(
+    isValidValue ? value : ""
```

Comment:
I would do another constat to this ternary above and just leave the variable inside this state

Reply:
Done